ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/vxlan-interface.j2

```diff
@@ -32,6 +32,20 @@
 {%     elif vxlan_interface.Vxlan1.vxlan.qos.map_dscp_to_traffic_class_decapsulation is arista.avd.defined(false) %}
 | Qos map dscp to traffic-class decapsulation | Disabled |
 {%     endif %}
+{%     if vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn is arista.avd.defined %}
+{%         if vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.interval is arista.avd.defined %}
+| Transmission rate set for remote VTEPs EVPN BFD configuration | {{ vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.interval }}ms |
+{%         endif %}
+{%         if vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.min_rx is arista.avd.defined %}
+| Expected minimum incoming rate set for remote VTEPs EVPN BFD configuration | {{ vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.min_rx }}ms |
+{%         endif %}
+{%         if vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.multiplier is arista.avd.defined %}
+| Multiplier set for remote VTEPs EVPN BFD configuration | {{ vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.multiplier }} |
+{%         endif %}
+{%         if vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.prefix_list is arista.avd.defined %}
+| Prefix-list reference for remote VTEPs EVPN BFD configuration | {{ vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.prefix_list }} |
```

Comment:
```suggestion
| Remote VTEPs EVPN BFD transmission rate | {{ vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.interval }}ms |
{%         endif %}
{%         if vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.min_rx is arista.avd.defined %}
| Remote VTEPs EVPN BFD expected minimum incoming rate (min-rx) | {{ vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.min_rx }}ms |
{%         endif %}
{%         if vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.multiplier is arista.avd.defined %}
| Remote VTEPs EVPN BFD multiplier  | {{ vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.multiplier }} |
{%         endif %}
{%         if vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.prefix_list is arista.avd.defined %}
| Remote VTEPs EVPN BFD prefix-list | {{ vxlan_interface.Vxlan1.vxlan.bfd_vtep_evpn.prefix_list }} |
```

I would even be partial to remove `Remote VTEPs EVPN` given that this is the only option for BFD in this context